### PR TITLE
Moffington Actually Maintains His PRs: The Tired First Episode

### PR DESCRIPTION
**Pull request by ReturnToZender**
## About The Pull Request
This does some misc. fixes to my PRs, particularly Waddle, to prevent them from throwing errors and make them work properly.

## Why It's Good For The Game
Bugfixes make the world go around

## Proof Of Testing
TBD

## Changelog

:cl:
fix: Fixed some indentation issues in tajaran.dm, adjusted some lines of examine code to prevent runtimes, borg names show up on the examine panel now instead of nothing
refactor: applies AddElementTrait to the waddling quirk, and mouse waddling
/:cl:

### CODE REVIEW

File: modular_skyrat/master_files/code/modules/mob/living/examine_tgui.dm [hunk 1]
```diff
@@ -75,18 +75,18 @@
 	// Now we handle silicon and/or human, order doesn't really matter
 	// If other variants of mob/living need to be handled at some point, put them here
 	if(issilicon(holder))
```
Comment (Cyprex) [inline modular_skyrat/master_files/code/modules/mob/living/examine_tgui.dm:77]:
```suggestion
	if(preferences && issilicon(holder))
```

Comment (Cyprex) [reply modular_skyrat/master_files/code/modules/mob/living/examine_tgui.dm:77]: do this instead of all the preferences? checks below

Comment (Cyprex) [reply modular_skyrat/master_files/code/modules/mob/living/examine_tgui.dm:77]: But also if preferences is null you might as well refuse to open the UI or something, this whole proc pretty much relies on preferences being not null

**Comment by Cyprex** [comment]
This is shitcode that needs to be fixed upstream. In particular here: https://github.com/Bubberstation/Bubberstation/pull/1482/files#diff-48e54ef0e9ffc02a1ed0d86526157da0c242352dd3e5a3a56d7d298d10d7e928R50 It grabs the preferences, and does take care of the client being null, but if so all it does is simply leave the preferences variable null.

**Comment by Majkl-J** [comment]
> This is shitcode that needs to be fixed upstream. In particular here: https://github.com/Bubberstation/Bubberstation/pull/1482/files#diff-48e54ef0e9ffc02a1ed0d86526157da0c242352dd3e5a3a56d7d298d10d7e928R50 It grabs the preferences, and does take care of the client being null, but if so all it does is simply leave the preferences variable null.

I can do that

File: modular_skyrat/master_files/code/modules/mob/living/examine_tgui.dm (hunk 1 above)
Comment (ReturnToZender) [reply modular_skyrat/master_files/code/modules/mob/living/examine_tgui.dm:77]: thanks boss

**Pull request merged**